runtime/src/main/scala/akka/grpc/scaladsl/play/AbstractAkkaGrpcServiceBindings.scala

```diff
@@ -0,0 +1,62 @@
+package akka.grpc.scaladsl.play
+
+import play.api.inject.Binding
+import play.api.{ Configuration, Environment, Logger }
+
+import scala.annotation.varargs
+import scala.reflect.ClassTag
+
+/**
+ * Helper for generating Play service bindings.
+ */
+abstract class AbstractAkkaGrpcServiceModule extends play.api.inject.Module {
+
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] =
+    services.flatMap(bindingsForService(_, environment, configuration))
+
+  protected def services: Seq[Class[_]]
+
+  @varargs // Helper for Java subclasses
+  final protected def classSeq(classes: Class[_]*): Seq[Class[_]] = classes
+
+  protected def bindingsForService[T](serviceClass: Class[T], environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
```

Comment:
This is how a service is bound. I try to be helpful in my error messages.

TODO: Write tests to check this works.